modules/apps/layout/layout-type-controller/layout-type-controller-test/src/testIntegration/java/com/liferay/layout/type/controller/asset/display/internal/portlet/test/AssetDisplayPageFriendlyURLProviderImplTest.java

```diff
@@ -109,6 +110,9 @@ public void testGetFriendlyURL() throws PortalException {
 	public void testGetFriendlyURLWithConfiguredURLSeparator()
 		throws Exception {
 
+		_setUpThemeDisplay(
```

Comment:
@bakayattila, for the future, it would be better to use getThemeDisplay and avoid using global _themeDisplay variable